config_defaults_inc.php

```diff
@@ -581,12 +581,10 @@
 $g_show_user_realname_threshold = NOBODY;
 
 /**
- * If use_x_priority is set to ON, what should the value be?
```

Comment:
With removal of this line, PHPdoc is missing a description of what this config is about